Sming/Arch/Esp8266/Core/HardwarePWM.cpp

```diff
@@ -23,115 +23,95 @@
  *
  */
 
+#include <HardwarePWM.h>
 #include <Clock.h>
 #include "ESP8266EX.h"
 #include <debug_progmem.h>
 
-#include <HardwarePWM.h>
-
 #define PERIOD_TO_MAX_DUTY(x) (x * 25)
 
 HardwarePWM::HardwarePWM(uint8_t* pins, uint8_t no_of_pins) : channel_count(no_of_pins)
 {
-	if(no_of_pins > 0) {
-		uint32_t io_info[PWM_CHANNEL_NUM_MAX][3];	// pin information
-		uint32_t pwm_duty_init[PWM_CHANNEL_NUM_MAX]; // pwm duty
-		for(uint8_t i = 0; i < no_of_pins; i++) {
-			io_info[i][0] = EspDigitalPins[pins[i]].mux;
-			io_info[i][1] = EspDigitalPins[pins[i]].gpioFunc;
-			io_info[i][2] = EspDigitalPins[pins[i]].id;
-			pwm_duty_init[i] = 0; // Start with zero output
-			channels[i] = pins[i];
-		}
-		const int initial_period = 1000;
-		pwm_init(initial_period, pwm_duty_init, no_of_pins, io_info);
-		update();
-		maxduty = PERIOD_TO_MAX_DUTY(initial_period); // for period of 1000
+	if(no_of_pins == 0) {
+		return;
+	}
+
+	uint32_t io_info[PWM_CHANNEL_NUM_MAX][3];	// pin information
+	uint32_t pwm_duty_init[PWM_CHANNEL_NUM_MAX]; // pwm duty
+	for(uint8_t i = 0; i < no_of_pins; i++) {
+		io_info[i][0] = EspDigitalPins[pins[i]].mux;
+		io_info[i][1] = EspDigitalPins[pins[i]].gpioFunc;
+		io_info[i][2] = EspDigitalPins[pins[i]].id;
+		pwm_duty_init[i] = 0; // Start with zero output
+		channels[i] = pins[i];
 	}
+	const int initial_period = 1000;
+	pwm_init(initial_period, pwm_duty_init, no_of_pins, io_info);
+	update();
+	maxduty = PERIOD_TO_MAX_DUTY(initial_period); // for period of 1000
 }
 
 HardwarePWM::~HardwarePWM()
 {
 	// There is no function in the SDK to stop PWM output, yet.
 }
 
-/* Function Name: getChannel
- * Description: This function is used to get channel number for given pin
- * Parameters: pin - Esp8266 pin number
- */
 uint8_t HardwarePWM::getChannel(uint8_t pin)
 {
 	for(uint8_t i = 0; i < channel_count; i++) {
 		if(channels[i] == pin) {
-			//debugf("getChannel %d is %d", pin, i);
+			// debug_d("getChannel %d is %d", pin, i);
```

Comment:
Remove commented debug statements.

Reply:
done

Reply:
is this really part of my PR? I have not touched the 8266 code

Reply:
That was me! See commit eaf77f91 - there's a new `getFrequency` method which is required.

Reply:
ah, I was really confused for a moment now. I have renamed the local vars to comply with camelCase now, not yet removed the debug calls. I think some of them are worthy of being there and functional, so I will uncomment them